cli/src/alluxio.org/cli/cmd/exec/test_journal_crash.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package exec
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+)
+
+var TestJournalCrash = &TestJournalCrashCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "testJournalCrash",
+		JavaClassName: "alluxio.cli.JournalCrashTest",
+		ShellJavaOpts: fmt.Sprintf(env.JavaOptFormat, env.ConfAlluxioLoggerType, "Console"),
+	},
+}
+
+type TestJournalCrashCommand struct {
+	*env.BaseJavaCommand
+	creates   string
+	deletes   string
+	maxAlive  string
+	renames   string
+	testDir   string
+	totalTime string
+}
+
+func (c *TestJournalCrashCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+func (c *TestJournalCrashCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use: "testJournalCrash [--creates <arg>] [--deletes <arg>] [--maxAlive <arg>]" +
```

Comment:
change the name to `journalCrashTest`

also remove the flag descriptions from here